python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -93,8 +97,10 @@ def start_ganga():
     logger.info("Checking Credentials")
 
     if missing_cred:
-        raise Exception("Failed due to missing credentials %s" %
-                        str(missing_cred))
+        raise Exception("Failed due to missing credentials %s" % str(missing_cred))
+
+    from Ganga.GPI import queues
+    queues.unlock()
```

Comment:
I want to make sure the queues system which was constructed in the GPI is unlocked.
This is the queues instance which is exposed to the user.

This should probably not be exported by ```exportToGPI(... ThreadPoolQueueMonitor()...)``` as in bootstrap. But this would require some work with singletons which is more hassle than it's worth imo to address this line.
(This would allow us to only start the queues up when they're needed would be nice :) ).